user_manual/room-testing-guide.adoc

```diff
@@ -0,0 +1,54 @@
+== Room Testing Procedure ==
+
+Each room needs to be tested for both audio and visual. In-room and recorded levels will be tested.
+
+There are several notes to remember during the each test:
+
+* Almost all settings are adjusted remotely, please be patient.
+* **Any** in-room change shall be as-directed from AV-NOC.
```

Comment:
Are you trying to say that they should only change things if directed? If so, it would be clearer to say "Do not change anything in-room unless directed by AV-NOC"